src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -277,90 +305,132 @@ const WorkspaceDashboard = _.flow(
-        ]),
-        h(InfoTile, { title: 'Google Project Id' }, [
-          div({ style: { display: 'flex' } }, [
-            h(TooltipCell, [googleProject]),
-            h(ClipboardButton, { text: googleProject, style: { marginLeft: '0.25rem' } })
+      div({ style: { paddingTop: '1rem' } }, [
+        div({ style: Style.dashboard.rightBoxContainer }, [
+          h(Collapse, {
+            title: h2({ style: Style.dashboard.collapsableHeader }, ['Workspace information']),
+            initialOpenState: workspaceInfoPanelOpen !== undefined ? workspaceInfoPanelOpen : true,
+            titleFirst: true,
+            onClick: () => setWorkspaceInfoPanelOpen(workspaceInfoPanelOpen === undefined ? false : !workspaceInfoPanelOpen)
+          }, [
+            h(InfoRow, { title: 'Last Updated' }, [new Date(lastModified).toLocaleDateString()]),
+            h(InfoRow, { title: 'Creation Date' }, [new Date(createdDate).toLocaleDateString()]),
+            h(InfoRow, { title: 'Workflow Submissions' }, [submissionsCount]),
+            h(InfoRow, { title: 'Access Level' }, [roleString[accessLevel]])
           ])
         ])
       ]),
-      div({ style: Style.dashboard.header }, ['Owners']),
-      _.map(email => {
-        return div({ key: email, style: { overflow: 'hidden', textOverflow: 'ellipsis' } }, [
-          h(Link, { href: `mailto:${email}` }, [email])
+      div({ style: { paddingTop: '1rem' } }, [
+        div({ style: Style.dashboard.rightBoxContainer }, [
+          h(Collapse, {
+            title: h2({ style: Style.dashboard.collapsableHeader }, ['Cloud information']),
+            initialOpenState: cloudInfoPanelOpen,
+            titleFirst: true,
+            onClick: () => setCloudInfoPanelOpen(!cloudInfoPanelOpen)
+          }, [
+            googleProject && h(InfoRow, { title: 'Cloud Name' }, [
+              h(GcpLogo, { title: 'Google Cloud Platform', role: 'img', style: { height: 16, width: 132, marginLeft: -15 } })
+            ]),
+            h(InfoRow, { title: 'Location' }, [bucketLocation ? h(Fragment, [
+              h(TooltipCell, [flag, ' ', regionDescription])
+            ]) : 'Loading...']),
+            h(InfoRow, { title: 'Google Project ID' }, [
+              h(TooltipCell, [googleProject]),
+              h(ClipboardButton, { text: googleProject, style: { marginLeft: '0.25rem' } })
+            ]),
+            h(InfoRow, { title: 'Bucket Name' }, [
+              h(TooltipCell, [bucketName]),
+              h(ClipboardButton, { text: bucketName, style: { marginLeft: '0.25rem' } })
+            ]),
+            Utils.canWrite(accessLevel) && h(InfoRow, {
+              title: 'Estimated Storage Cost',
+              subtitle: storageCostEstimate ? `Updated on ${new Date(storageCostEstimateUpdated).toLocaleDateString()}` : 'Loading last updated...'
+            }, [storageCostEstimate || '$ ...']),
+            Utils.canWrite(accessLevel) && h(InfoRow, { title: 'Bucket Size' }, [bucketSize]),
```

Comment:
Should the last updated time for the bucket size also be shown here?

Reply:
Good question- I'll check with Lou on Tuesday

Reply:
Added in a8c9e15e2570b00c02d8ac385cf2df0d7825b843